docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -777,8 +777,12 @@ The default is `8008`.
     network: "mainnet"
     ```
 
-Predefined network configuration. Accepts a predefined network name, or file path or URL
-to a YAML configuration file. The default is `mainnet`.
+Predefined network configuration.
+Accepts a predefined network name, or file path or URL to a YAML configuration file or directory.
+Directories must hold a `phase0.yaml` file and can hold an optional `altair.yaml` file.
+See the [MainNet resources directory] for an example.
```

Comment:
I'd suggest just cutting this line.  Can't think of a resource to link to that is guaranteed to be stable and correct.